$OpenBSD: patch-configure_ac,v 1.4 2013/04/01 09:29:12 jasper Exp $

From 42da25a17292f925046efd630d39fb7127c268d9 Mon Sep 17 00:00:00 2001
From: Jasper Lievisse Adriaanse <jasper@humppa.nl>
Date: Tue, 26 Mar 2013 15:05:49 +0000
Subject: Unbreak configure

- set a proper version for the required libxml
- make sure the variable gets expanded.

--- configure.ac.orig	Tue Mar 26 15:58:28 2013
+++ configure.ac	Tue Mar 26 16:04:46 2013
@@ -32,7 +32,7 @@ GTK_REQUIRED=3.0.0
 GTHREAD_REQUIRED=2.12
 ICON_THEME_REQUIRED=2.20
 GTKSOURCEVIEW_REQUIRED=3.2.0
-LIBXML_REQUIRED=2.24
+LIBXML_REQUIRED=2.0
 
 AC_MSG_CHECKING([for Win32])
 case "$host" in
@@ -49,14 +49,14 @@ AM_CONDITIONAL(OS_WIN32, [test $os_win32 = yes])
 PKG_CHECK_MODULES(LIBGSSDP, gssdp-1.0 >= $GSSDP_REQUIRED)
 PKG_CHECK_MODULES(LIBGUPNP, gupnp-1.0 >= $GUPNP_REQUIRED)
 PKG_CHECK_MODULES(LIBSOUP, libsoup-2.4 >= $SOUP_REQUIRED)
-PKG_CHECK_MODULES(LIBXML, libxml-2.0 >= LIBXML_REQUIRED)
+PKG_CHECK_MODULES(LIBXML, libxml-2.0 >= $LIBXML_REQUIRED)
 PKG_CHECK_MODULES(GTK, gtk+-3.0 >= $GTK_REQUIRED)
 PKG_CHECK_MODULES(GTHREAD, gthread-2.0 >= $GTHREAD_REQUIRED)
 PKG_CHECK_MODULES(GNOME_ICON_THEME, gnome-icon-theme >= $ICON_THEME_REQUIRED)
 PKG_CHECK_MODULES(GIO, glib-2.0 >= $GLIB_REQUIRED
                        gobject-2.0 >= $GLIB_REQUIRED
                        gio-2.0 >= $GLIB_REQUIRED)
-PKG_CHECK_MODULES(GTKSOURCEVIEW, gtksourceview-3.0 >= GTKSOURCEVIEW_REQUIRED,
+PKG_CHECK_MODULES(GTKSOURCEVIEW, gtksourceview-3.0 >= $GTKSOURCEVIEW_REQUIRED,
                   AC_DEFINE([HAVE_GTK_SOURCEVIEW], [1], [Use GtkSourceView])
                   AC_SUBST([TEXT_VIEW],[GtkSourceView]),
                   AC_SUBST([TEXT_VIEW],[GtkTextView]))
